src/main/java/com/github/pop4959/srbot/command/CommandChart.java

```diff
@@ -25,49 +25,27 @@ public void execute(MessageReceivedEvent event, String[] args) {
         if (args.length < 1) {
             event.getChannel().sendMessage(LANGUAGE.get("noId")).queue();
             return;
-        } else if (args.length < 2) {
-            event.getChannel().sendMessage(LANGUAGE.get("noSeason")).queue();
-            return;
         }
 
         String id = Utils.resolveID64(new SteamUser(Main.getClient()), args[0]);
-        if (id == null) {
-            event.getChannel().sendMessage(LANGUAGE.get("wrongId")).queue();
-            return;
-        }
-
-        String[] seasons = {"off", "beta", "winter", "christmas"};
-
-        int season = -1;
-        boolean isInt = true;
-
-        try {
-            season = Integer.parseInt(args[1]);
-        } catch (NumberFormatException e) {
-            isInt = false;
-        }
-
-        if (!isInt) {
-            for (int i = 0; i < 4; ++i) {
-                if (seasons[i].toLowerCase().equals(args[1].toLowerCase())) {
-                    season = ++i;
-                    break;
-                }
-            }
-        }
+        Integer season = args.length < 2 ? Integer.valueOf(1) : Utils.getSeason(args[1]);
 
-        if (season < 1 || season > 4) {
-            event.getChannel().sendMessage(LANGUAGE.get("wrongSeason")).queue();
+        if (!Utils.checkSeason(event, id, season)) {
             return;
         }
 
         event.getChannel().sendTyping().queue();
 
         String output;
         try {
-            Process chart = Runtime.getRuntime().exec(String.format("node scripts/chart.js %s %s", id, season));
+            Process chart = Runtime.getRuntime().exec(String.format("node ../scripts/chart.js %s %s", id, season));
```

Comment:
Why was the path changed here? It should still be in the same directory as before. This will probably cause the script not to run correctly.

Reply:
eh again my working directory is different from yours. will update in a few

Reply:
The gradle build script should be moving the jar into the bot folder. That is the intended working directory.

Reply:
My layout is exactly like yours yet I get a different working directory?

The path was updated.